src/files.ts

```diff
@@ -9,8 +9,10 @@ import { DOT, DOTFILE } from './dotfile';
 import { ERROR, LOG, checkIfOnline, getAPIFileType, getProjectSettings, logError, spinner } from './utils';
 
 const ts2gas = require('ts2gas');
+// TODO: include ts2gas .d.ts in package
```

Comment:
Can we do this now? What needs to change?

Reply:
@grant currently grant/ts2gas has:

[x] tsconfig.json with `declaration` set to `true`
[x] package.json with `types` set to `./src/index.d.ts`
[ ] `.d.ts` (and optionaly `.ts` included in the published package

Last point is because of `.npmignore` which excludes any `*.ts`

If you want, I could issue a new PR (from a `branch`) with the `.npmignore` change, and possibly the additionnal changes:

- `src` folder contains only the source `.ts`
- a new `lib` folder which will contains `.js` and `.d.ts``
- `package.json` attributes `main` and `types` to properly point to the `lib` folder

Reply:
See https://github.com/grant/ts2gas/issues/39

Let's fix it and publish the `.d.ts`.

Reply:
This could wait for a future PR IMHO. For an effective change, we need a new `ts2gas` release with updated dependency in `package.json`